src/main/java/org/fundacionjala/trello/pages/menus/MenuBoards.java

```diff
@@ -0,0 +1,67 @@
+package org.fundacionjala.trello.pages.menus;
+
+import org.fundacionjala.trello.pages.board.BoardClosed;
+import org.fundacionjala.trello.pages.board.BoardForm;
+import org.fundacionjala.trello.pages.board.BoardPage;
+import org.fundacionjala.trello.pages.core.WebObject;
+import org.fundacionjala.trello.pages.forms.FormPage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public final class MenuBoards extends WebObject {
+
+    private static final String FIND_BOARDS = "input[data-test-id='header-boards-menu-search']";
+    private static final String CREATE_BOARD = "button[data-test-id='header-boards-menu-create-board']";
+    private static final String CLOSED_BOARDS = "button[data-test-id='header-boards-menu-open-closed']";
+
+    @FindBy(css = FIND_BOARDS)
+    private WebElement findBoards;
+
+    @FindBy(css = CREATE_BOARD)
+    private WebElement createBoardButton;
+
+    @FindBy(css = CLOSED_BOARDS)
+    private WebElement closedBoardsButton;
+
+    public MenuBoards(final WebDriver driver) {
+        super(driver);
+    }
+
+    /**
+     * Indicates if the actual page is displayed.
+     *
+     * @return true if actual page is displayed, else false.
+     */
+    @Override
+    public boolean isDisplayed() {
+        return findBoards.isDisplayed();
+    }
+
+    public BoardPage goToBoardOnPersonalBoards(final String boardName) {
+        WebElement boardElement = driver.findElement(By
```

Comment:
The method name is too long, I suggest you change it for a shorter one as openPersonalBoard()